docs/4.0/components/modal.md

```diff
@@ -45,8 +45,8 @@ Below is a _static_ modal example (meaning its `position` and `display` have bee
           <p>Modal body text goes here.</p>
         </div>
         <div class="modal-footer">
-          <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
           <button type="button" class="btn btn-primary">Save changes</button>
+          <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
```

Comment:
leave the order here as it is originally, but change the order in the `{% highlight html %}` that comes immediately after instead. the live example is right/consistent with the rest of the page, it's the visible highlighted html code that's wrong

Reply:
Thank you for the input! I'll try to do it now.

Reply:
The test seems to be failing though.